test/services/CachingNpsServiceTest.scala

```diff
@@ -28,12 +30,18 @@ import uk.gov.hmrc.play.test.UnitSpec
 
 import scala.concurrent.Future
 
-class CachingNpsServiceTest extends UnitSpec with ScalaFutures with MockitoSugar {
+class CachingNpsServiceTest extends UnitSpec with ScalaFutures with MockitoSugar with BeforeAndAfterEach {
```

Comment:
This unit test is failing. Can you please have a look?

Reply:
Added integrationPatience